en/config/actuators.md

```diff
@@ -109,31 +146,104 @@ Selecting these requires a reboot before they are applied.
 
 ## Actuator Testing
 
-<!-- Not documented: "Identify and assign motors" -->
- 
-When testing actuators, make sure that:
-- Motors spin at the "minimum thrust" position.
+### Identify & Assign Motors
 
-  The sliders snap into place at the lower end, and motors are turned off (disarmed).
-  The "minimum thrust" position is the next slider position, which commands the minimum thrust.
-  For PWM motors, adjust the minimum output value such that the motors spin at that slider position (not required for DShot).
-  :::note
-  VTOLs will automatically turn off motors pointing upwards during fixed-wing flight.
-  For Standard VTOLs these are the motors defined as multicopter motors.
-  For Tiltrotors these are the motors that have no associated tilt servo.
-  Tailsitters use all motors in fixed-wing flight.
-  :::
-- Servos move into the direction of the convention described above.
-  :::note
-  A trim value can be configured for control surfaces, which is also applied to the test slider.
-  :::
+![](../../assets/config/actuators/identify_motors_button.png)
+
+QGC Supports a feature of identifying the motors by sending the motor command to different output pins and letting you click on the motor to map the Motor to each Output ports.
+
+To use this mode:
+1. Setup the Motors Geometry correctly to include all the motors in your vehicle you want to configure
```

Comment:
Add: 2. select the output tab where you want to assign the motors, if not assigned already

Reply:
I might revisit this a bit later, but for now adding the point.,

```suggestion
1. Setup the Motors Geometry correctly to include all the motors in your vehicle you want to configure
1. Select the Output tab where you want to assign the motors (if not assigned already)
```